sklearn/linear_model/least_angle.py

```diff
@@ -587,14 +587,15 @@ class Lars(LinearModel, RegressorMixin):
     sklearn.decomposition.sparse_encode
 
     """
+    method = 'lar'
```

Comment:
It's not wonderful, but I don't think it's this PR's job to fix it. The problem derives from the use of inheritance, but non-use of `super...__init__`. In the current code, `method` is twice set, like this, as a class attribute (in the CV objects), and twice as an instance attribute. It might make more sense to have it as a private attribute, but this approach at least ensures a consistency that is not in master, and that the instance's `__dict__` only has true parameters.